pkg/console/controllers/healthcheck/controller.go

```diff
@@ -189,3 +210,10 @@ func clientWithCA(caPool *x509.CertPool) *http.Client {
 		},
 	}
 }
+
```

Comment:
Here I wasnt really sure how to check the `count(compute nodes) < router replicas` which @wking was mentioning in his [msg](https://redhat-internal.slack.com/archives/C6A3NV5J9/p1700684105271709?thread_ts=1700670355.775489&cid=C6A3NV5J9).